content/docs/custom-mounting.md

```diff
@@ -9,8 +9,14 @@ You can now provide your own element for Decap CMS to mount in by setting the ta
 
 This is useful if you want to create a wrapper around the CMS, like a custom header, footer, or sidebar.
 
+Make sure to load decap javascript after document (DOM) is ready, so that it can detect existing element with proper id.
```

Comment:
Grammatically better:

Make sure to load the Decap script after the DOM is ready so that it can detect existing elements with proper ID.